.swiftformat

```diff
@@ -0,0 +1,8 @@
+--closingparen same-line
+--commas inline
+--exclude Examples
+--header "{file} - Copyright {year} SwifterSwift"
+--ifdef no-indent
+--importgrouping testable-top
+--maxwidth 120
+--swiftversion 5.1
```

Comment:
Aren't we on 5.2?

Reply:
The podspec said 5.1, but I think we are building 5.2. I'll change it.